cmd/zoekt-sourcegraph-indexserver/cleanup.go

```diff
@@ -0,0 +1,154 @@
+package main
+
+import (
+	"log"
+	"os"
+	"path/filepath"
+	"sort"
+	"time"
+
+	"github.com/google/zoekt"
+)
+
+func cleanup(indexDir string, repos []string, now time.Time) {
+	trashDir := filepath.Join(indexDir, ".trash")
+	if err := os.MkdirAll(trashDir, 0755); err != nil {
+		log.Printf("failed to create trash dir: %v", err)
+	}
+
+	trash := getShards(trashDir)
+	index := getShards(indexDir)
+
+	// trash: Remove old shards and conflicts with index
+	minAge := now.Add(-24 * time.Hour)
+	for repo, shards := range trash {
+		old := false
+		for _, shard := range shards {
+			if shard.ModTime.Before(minAge) {
```

Comment:
```suggestion
			if shard.ModTime.Before(minAge) || shard.ModTime.Equal(minAge) {
```

Reply:
the next run of cleanup will remove it. I think its probably clearer to just stick with what it has, or use a better name than `minAge`?

Reply:
> the next run of cleanup will remove it. 

Oh, yes, it's fine then!